dev_scripts/env.py

```diff
@@ -123,8 +121,13 @@
 WORKDIR /home/user
 VOLUME /home/user/dangerzone
 
+# Install Poetry under ~/.local/bin.
+# See https://github.com/freedomofpress/dangerzone/issues/351
+RUN pipx install poetry
+ENV PATH="$PATH:/home/user/.local/bin"
```

Comment:
The path thing doesn't seem to be need on fedora. So I'd try like this:

```suggestion
RUN pipx install poetry
```

Reply:
Good catch, thanks.

Reply:
Oh, wait though. This Dockerfile snippet is shared across all of our distros, even the non-Fedora ones. It's best to keep it, so that we don't make these snippets even less readable :-/.